File: program/convert-fahrenheit-to-kelvin/convert-fahrenheit-to-kelvin.py
def fahrenheit_to_kelvin(temp_f):
    try:
        temp_k = (temp_f - 32) * 5 / 9 + 273.15
        return temp_k
    except TypeError as e:
        print(f"Error: The input temperature must be a numerical value. {e}")
    except Exception as e:
        print(f"An unexpected error occurred. {e}")


if __name__ == "__main__":
    fahrenheit = int(input("Fahrenheit input: "))
    print("Kelvin:", fahrenheit_to_kelvin(fahrenheit))

File: program/program/convert-fahrenheit-to-kelvin/ConvertFahrenheitToKelvin.swift
func fahrenheitToKelvin(fahrenheit: Double) -> Double {
    let kelvin = (fahrenheit + 459.67) * 5/9
    return kelvin
}

print("Enter temperature in Fahrenheit: ")
if let input = readLine(), let fahrenheit = Double(input) {
    let kelvin = fahrenheitToKelvin(fahrenheit: fahrenheit)
    print("Temperature in Kelvin: \(kelvin)")
} else {
    print("Invalid input. Please enter a valid temperature in Fahrenheit.")
}
